packages/did-session/src/index.ts

```diff
@@ -321,6 +327,45 @@ export class DIDSession {
     return didWithCap
   }
 
+  /**
+   * Get a session for the given accountId, if one exists, otherwise creates a new one.
+   */
+  static async get(
+    account: AccountId,
+    authMethod: AuthMethod,
+    authOpts: AuthOpts = {}
+  ): Promise<DIDSession> {
+    if (typeof window === 'undefined')
```

Comment:
Could this be moved to the `SessionStore` logic?
Would be good if we could use `globalThis` instead of `window` to access the global object, that would make the code work in workers as well.

Reply:
Good point!